src/Confluent.Kafka/SysLogLevel.cs

```diff
@@ -3,39 +3,39 @@ namespace Confluent.Kafka
     /// <summary>
     ///  Represents enumeration with levels comming from syslog(3)
     /// </summary>
-    public enum LogLevel
+    public enum SysLogLevel
```

Comment:
nit: Should probably be "SyslogLevel", since syslog is referred to as "syslog", not "SysLog".